tremor-pipeline/src/metrics.rs

```diff
@@ -74,3 +74,25 @@ pub fn value(
         TIMESTAMP: timestamp
     })
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
how is this related?

Reply:
It's not, I don't know how this got here :joy: